.eslintrc.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
             'error',
             {
                 ignoreRestSiblings: true,
+                varsIgnorePattern: '^_',
```

Comment:
Now you can do `const [_, setUser] = React.useState();` without eslint error

Reply:
Shouldn't it be `^_$` to prevent variables beginning with `_` ?

Reply:
I saw some people wire _ignore or sutff like that. I prefer the underscore only too